scripts/nb-tester/qiskit_docs_notebook_tester/__init__.py

```diff
@@ -225,6 +225,12 @@ async def execute_notebook(path: Path, config: Config) -> bool:
     print(f"✅ No problems in {path} (written)")
     return True
 
+async def _execute_in_kernel(kernel, code: str) -> None:
```

Comment:
```suggestion
async def _execute_in_kernel(kernel: AsyncKernelClient, code: str) -> None:
```

This is imported `from jupyter_client.manager`